### DISCUSSION TITLE
battery: support for state "Not charging"

### DISCUSSION THREAD

**Discussion by visadb** [polybar/polybar#2737] [Ideas]
TL;DR: the battery module interprets the `Not charging` state as discharging which I find less than ideal.

I run my laptop on AC power most of the time, so to protect my battery, I've set a charge limit (`echo 70 > /sys/class/power_supply/BAT0/charge_control_end_threshold`). At 70% battery or above, my battery stops charging and the laptop effectively runs on AC only, and `/sys/class/power_supply/BAT0/status` reads `Not charging`. The battery module interprets this as discharging which is not accurate: I'd like to be able to show a different label in this state.

To cover this case, I propose adding a new state `not-charging` to complement the current states `low`, `discharging`, `charging`, and `full`.
